### ISSUE TITLE
PhotoView.customChild doesn't always work correctly with Image.network

### ISSUE THREAD

**Issue by androidmitry** [fireslime/photo_view#391]
**Describe the bug**

`PhotoView.customChild` with `Image.network` doesn't show image if `Image.network` has `loadingBuilder`

**To Reproduce**
Try this sample code:
```
import 'package:flutter/material.dart';
import 'package:photo_view/photo_view.dart';

String imageUrl =
    "https://www.wallpapers13.com/wp-content/uploads/2016/01/Beautiful-lake-mountain-forest-desktop-wallpapers.jpg";

class DemoPage extends StatelessWidget {
  @override
  Widget build(BuildContext context) {
    return Scaffold(
      appBar: AppBar(
        title: Text('Zoom image demo'),
      ),
      body: PhotoView.customChild(
        child: Image.network(
          imageUrl,
          loadingBuilder: (BuildContext context, Widget child, ImageChunkEvent loadingProgress) {
            return Center(
              child: CircularProgressIndicator(),
            );
          }
        ),
      )
    );
  }
}

```

**What is the current behavior?**
If `Image.network` has `loadingBuilder` loading indicator never disappears. If I remove `loadingBuilder` image is displayed correctly.

**Expected behavior**
Display image when it is loaded.

**Which versions of Flutter/Photo View, and which browser / OS are affected by this issue? Did this work in previous**
`photo_view: ^0.10.3`
`flutter: 1.22.6`
Tested on Android emulator and on real Android device

**Comment by progid** [comment]
Have the same problem, but I use PhotoViewGallery.builder

**Comment by renancaraujo** [comment]
1- I am not sure why this is working without photo_view, but your usage of **Image**s `loadingBuilder` is wrong. According to the [docs](https://api.flutter.dev/flutter/widgets/Image/loadingBuilder.html), loading builder must return `child` upon completion. 
2- Is there any reason to not use the build int loading builder that photo_view offers?

**Comment by progid** [comment]
I'm using loadingBuilder inside PhotoViewGallery.builder

**Comment by androidmitry** [comment]
1 - Good remark, returning child solves the issue, thanks. Valid loadBuilder will look something like this:
```
loadingBuilder: (BuildContext context, Widget child, ImageChunkEvent loadingProgress) {
            if(loadingProgress == null)
              return child;

            return Center(
              child: CircularProgressIndicator(),
            );
          }
```
2 - the reason of not using build in loading builder is because of another issue. In 0.10.3 [errorBuilder is never called](https://github.com/fireslime/photo_view/issues/337). This issue is solved in 0.11.0+, but you must use dark SDK 2.12 or later.
